app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -17,6 +17,19 @@ const Notes = () => {
   const [showDeleteAlert, setShowDeleteAlert] = useState(false);
   const [searchTerm, setSearchTerm] = useState("");
   const [showMenu, setShowMenu] = useState(false);
+  const [notesList, setNotes] = useState(notes);
+  const [selectedNoteId, setSelectedNoteId] = useState();
```

Comment:
This is a bad practice. Don't declare states without providing a default value signifying what kind of data the state can hold.